README.md

```diff
@@ -2,15 +2,15 @@
 
 This is the working area for the IETF [RESUMABLE Working Group](https://datatracker.ietf.org/wg/resumable/documents/) Internet-Draft, "tus - Resumable Uploads Protocol".
 
-* [Editor's Copy](https://tus.github.io/tus-v2/#go.draft-tus-httpbis-resumable-uploads-protocol.html)
-* [Working Group Draft](https://tools.ietf.org/html/draft-tus-httpbis-resumable-uploads-protocol)
-* [Compare Editor's Copy to Working Group Draft](https://tus.github.io/tus-v2/#go.draft-tus-httpbis-resumable-uploads-protocol.diff)
+* [Editor's Copy](https://tus.github.io/tus-v2/#go.draft-ietf-httpbis-resumable-upload.html)
+* [Working Group Draft](https://tools.ietf.org/html/draft-ietf-httpbis-resumable-upload)
+* [Compare Editor's Copy to Working Group Draft](https://tus.github.io/tus-v2/#go.draft-ietf-httpbis-resumable-upload.diff)
```

Comment:
These three are not strictly necessary, but might help maintain consistency. The diff will be broken until draft-ietf-httpbis-resumable-upload-00 is uploaded to the datatracker.